app/code/Magento/Catalog/Controller/Adminhtml/Product/Action/Attribute/Save.php

```diff
@@ -32,23 +33,50 @@ class Save extends \Magento\Catalog\Controller\Adminhtml\Product\Action\Attribut
     protected $_catalogProduct;
 
     /**
-     * @var \Magento\CatalogInventory\Api\Data\StockItemInterfaceFactory
-     */
-    protected $stockItemFactory;
-
-    /**
      * Stock Indexer
      *
      * @var \Magento\CatalogInventory\Model\Indexer\Stock\Processor
      */
     protected $_stockIndexerProcessor;
 
     /**
+     * @var \Magento\CatalogInventory\Api\Data\StockItemInterfaceFactory
+     */
+    protected $stockItemFactory;
+
+    /**
      * @var \Magento\Framework\Api\DataObjectHelper
      */
     protected $dataObjectHelper;
 
     /**
+     * @var \Magento\Framework\Stdlib\DateTime\TimezoneInterface
+     */
+    protected $localeDate;
+
+    /**
+     * Eav config
+     *
+     * @var \Magento\Eav\Model\Config
+     */
+    protected $eavConfig;
```

Comment:
please don't use protected access modifier for newly created attributes and methods .

use private instead